content/shmem_fxor.tex

```diff
@@ -0,0 +1,45 @@
+\apisummary{
+  Atomically perform a fetching bitwise exclusive or operation on a
+  remote data object.
+}
+
+\begin{apidefinition}
+
+\begin{C11synopsis}
+TYPE shmem_fxor(TYPE *dest, TYPE value, int pe);
+\end{C11synopsis}
+where \TYPE{} is one of the standard \ac{AMO} types specified by
+Table~\ref{stdamotypes}.
+
+\begin{Csynopsis}
+TYPE shmem_<TYPENAME>_fxor(TYPE *dest, TYPE value, int pe);
+\end{Csynopsis}
+where \TYPE{} is one of the standard \ac{AMO} types and has a corresponding
+\TYPENAME{} specified by Table \ref{stdamotypes}.
+
+\begin{apiarguments}
+
+  \apiargument{OUT}{dest}{A pointer to the remotely accessible data object to
+    be updated.}
+  \apiargument{IN}{value}{The operand to the bitwise atomic operation.}
```

Comment:
"bitwise atomic operation" seems a bit lazy.  Can we replace this with "bitwise XOR operation"